src/Constraints/IsoCodesGenericValidator.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace SLLH\IsoCodesValidator\Constraints;
+
+use IsoCodes\IsoCodeInterface;
+use SLLH\IsoCodesValidator\AbstractIsoCodesConstraintValidator;
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+class IsoCodesGenericValidator extends AbstractIsoCodesConstraintValidator
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function validate($value, Constraint $constraint)
+    {
+        parent::validate($value, $constraint);
+
+        $constraintClass = str_replace('SLLH\\IsoCodesValidator\\Constraints\\', '', get_class($constraint));
+        /** @var IsoCodeInterface $isoCodesClass */
```

Comment:
this phpdoc is wrong. The variable ``$isoCodesClass`` contains a string, not a ``IsoCodeInterface``